app/pages/about/team-page.cjsx

```diff
@@ -187,6 +190,13 @@ teamMembers =
     bio: counterpart "team.content.andreaSimenstad.bio"
     image: "/assets/team/andrea.jpg"
     location: "minnesota"
+  beverleyNewing:
+    name: "Beverley Newing"
+    twitter: "beverleynewing"
+    title: counterpart "team.content.beverleyNewing.title"
+    bio: counterpart "team.content.beverleyNewing.bio"
+    image: "/assets/team/bev.jpg"
```

Comment:
can you make this filename a little more explicit, i.e. `beverley.jpg`. I read that with more information...naming things is hard.